pkgs/applications/audio/pd-plugins/gem/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv
+, fetchFromGitHub
+, fetchpatch
+, autoreconfHook
+, libtool
+, pkg-config
+, puredata
+, libGL
+, libGLU
+, glew
+, freeglut
+, libv4l
+, libX11
+, file
+ }:
+
+stdenv.mkDerivation rec {
+  pname = "gem";
+  # The patch below applies to the latest release (v0.94), but then the build
+  # fails. I didn't track down what changed between that version and the
+  # current master that fixes the build on Nix
+  version = "unstable-2020-03-26";
```

Comment:
Version [must start with a digit](https://nixos.org/nixpkgs/manual/#sec-package-naming), the "unstable" should go into `pname`.

Reply:
thx, fixed